cclient/serialization/serializer.h

```diff
@@ -0,0 +1,135 @@
+// Copyright 2018 IOTA Foundation
+
+#ifndef CCLIENT_SERIALIZATION_SERIALIZER_H
+#define CCLIENT_SERIALIZATION_SERIALIZER_H
+
+#include <stdlib.h>
+#include "request/requests.h"
+#include "response/responses.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct serializer_base serializer;
+
+typedef struct {
```

Comment:
could we name these `requestname_(de)serialize_get_size`?
allows for better alphabetical sorting :D

Reply:
done